BridgeAppSDK/SBAProfileDataSource.swift

```diff
@@ -33,6 +33,9 @@
 
 import Foundation
 
+public var SBAProfileJSONFilename = "Profile"
+public var SBAProfileDataSourceClassType = "ProfileDataSource"
```

Comment:
...was assuming they'd essentially act like globals; making them var allows folks to change them before they get used so they load from differently-named files without modifying any code--not a huge gain, could switch to let if you prefer